ere-guests/stateless-validator/sp1/src/main.rs

```diff
@@ -46,3 +47,38 @@ fn init_tracing_just_like_println() {
         .with_max_level(tracing::Level::INFO) // capture info! and up
         .init(); // set as global default
 }
+
+fn from(fork_spec: ForkSpec) -> ChainSpec {
+    let spec_builder = ChainSpecBuilder::default()
+        .chain(Chain::mainnet())
+        .genesis(Genesis::default());
```

Comment:
For single-block stateless validation, this sounds okay -- but I still would like a final check from someone from Reth.